polymer/src/sample-detail.html

```diff
@@ -0,0 +1,176 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../node_modules/@npm-polymer/polymer/polymer-element.html">
+<link rel="import" href="../node_modules/@npm-polymer/app-route/app-route.html">
+<link rel="import" href="../node_modules/@npm-polymer/iron-flex-layout/iron-flex-layout.html">
+
+<link rel="import" href="amp-my-embed.html">
+<link rel="import" href="sample-category-data.html">
+
+<dom-module id="sample-detail">
+
+  <template>
+
+    <style>
+
+      :host {
+        display: block;
+      }
+
+      #content {
+        @apply --layout-horizontal;
+        @apply --layout-center-justified;
+      }
+
+      h1 {
+        font-size: 24px;
+        font-weight: 500;
+        line-height: 28px;
+        margin: 0;
+      }
+
+      @media (max-width: 767px) {
+
+        #content {
+          @apply --layout-vertical;
+          @apply --layout-center;
+        }
+
+        .detail {
+          box-sizing: border-box;
+          margin: 32px 0;
+          padding: 0 24px;
+          width: 100%;
+          max-width: 600px;
+        }
+
+        h1 {
+          font-size: 20px;
+          line-height: 24px;
+        }
+
+      }
+
+    </style>
+
+    <!--
+      app-route provides the name of the category and the item.
+    -->
+    <app-route
+        route="[[route]]"
+        pattern="/:category/:item"
+        data="{{routeData}}"></app-route>
+
+    <!--
+      sample-category-data provides the item data for a given category and item name.
+    -->
+    <sample-category-data
+        id="categoryData"
+        category-name="[[routeData.category]]"
+        category="{{category}}"
+        item-name="[[routeData.item]]"
+        item="{{item}}"
+        failure="{{failure}}"></sample-category-data>
+
+    <header>
+          <h1>[[item.title]]</h1>
+    </header>
+
+    <amp-my-embed
```

Comment:
Can you find a better name than amp-my-embed? Maybe: amp-page

Reply:
replaced